### ISSUE TITLE
Check file content

### ISSUE THREAD

**Issue by JuliaSprenger** [INT-NIT/AnDO#72]
Currently the AnDOChecker only considers folder and file names, but does not check the content of metadata files for required fields.
We should discuss on how this kind of check can be integrated in the current architecture. These checks need to be quite flexible, as some metadata files or entries are only required in certain circumstances, eg. a `tasks.csv` is only required if the concept of tasks is used in the dataset.

One possible implementation would be function / method based, i.e. having one function checking the consistency of all `task` entries within the structure. However, this would be a quite different approach to the current `rule_set` approach taken for file and folder names.

Are there any additional proposals on how to add these specifics checks to the current architecture?

**Comment by SylvainTakerkart** [comment]
Yes, but I have to admit I'm a bit reluctant to get into this... Could we first make progress on the DigLab2AnDO development? And get it to be used by users...

Also, we might want to discuss this, this might be easier...

**Comment by Remi-Gau** [comment]
Yes I would keep that for later too.
